_data/authors.yml

```diff
@@ -14,6 +14,13 @@ isla_staden:
   avatar: IS_whitakerblogprofile.jpg
   twitter: islastaden
 
+ang_li: # No spaces please
+  name: Ang Li # Your name
+  email: ali@turing.ac.uk # Your email
+  bio: # Add a few words about yourself within the quotation marks.
```

Comment:
Hi @sprak0323 - please can you submit a pull request to the `kw-edit` branch with a short bio in this space, and a link to a picture in the images folder. Let me know if you're not sure how to do this!